README.md

```diff
@@ -106,9 +106,12 @@ _Additional information_
   * An array list of columns to bucket data, ignored if using Iceberg
 * `bucket_count` (`default=none`)
   * The number of buckets for bucketing your data, ignored if using Iceberg
+* `table_type` (`default='hive'`)
+  * The type of table
+  * Supports `hive` or `iceberg`
 * `format` (`default='parquet'`)
   * The data format for the table
-  * Supports `ORC`, `PARQUET`, `AVRO`, `JSON`, `TEXTFILE` or `iceberg`
+  * Supports `ORC`, `PARQUET`, `AVRO`, `JSON`, `TEXTFILE`
```

Comment:
Somehow I suggest to add a clarification note that in Iceberg we can choose the `format` under the hood, I know that is obvious, but I think that being explicit avoids confusion.